config/routes.rb

```diff
@@ -1,10 +1,19 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   resources :genres
-  resources :movies
   resources :users
   
+  get '/movies', to: "movies#index"
+  get '/movies/:id', to: "movies#show"
+  put '/movies/:movie_id/lists/:list_id', to:'movies#add_movie_to_a_list'
```

Comment:
How about putting this under `lists_controller` instead of `movies_controller`? Since you're actually modifying a list, not a movie. And that way we can rename this method as `add_movie`